### PR TITLE
Updated Views for Tailwindcss v1

### PR DESCRIPTION
**Pull request by rverrips**
Dear Michael

I need to go back and review this myself and finalise everything (including the README) once tailwind v1 is released.

Note: laravel-mix-tailwindcss also needs to be updated (or manually adjusted) at present - change node_modules/laravel-mix-tailwind/index.js tailwind.js to tailwind.config.js 

Please review and advise if I missed anything, which is highly likely.

Rock on down-unda!

### CODE REVIEW

File: src/tailwindcss-stubs/resources/views/auth/passwords/reset.blade.php [hunk 1]
```diff
@@ -16,43 +16,43 @@
...
                                 {{ __('Confirm Password') }}:
                             </label>
 
-                            <input id="password-confirm" type="password" class="shadow appearance-none border rounded w-full py-2 px-3 text-grey-darker leading-tight focus:outline-none focus:shadow-outline" name="password_confirmation" required>
+                            <input id="password-confirm" type="password" class="shadow appearance-none border rounded w-full py-2 px-3 text-gray-700 leading-tight focus:outline-none focus:shadow-outline" name="password_confirmation" required>
                         </div>
 
                         <div class="flex flex-wrap">
-                            <button type="submit" class="bg-blue hover:bg-blue-dark text-white font-bold py-2 px-4 rounded focus:outline-none focus:shadow-outline">
+                            <button type="submit" class="bg-blue-500 hover:bg-blue-700 text-gray-100 font-bold font-bold py-2 px-4 rounded focus:outline-none focus:shadow-outline">
```
Comment (imliam) [inline src/tailwindcss-stubs/resources/views/auth/passwords/reset.blade.php]: Seem to have mistakenly doubled up on the `font-bold` class here.

Comment (rverrips) [reply src/tailwindcss-stubs/resources/views/auth/passwords/reset.blade.php]: Good catch, thanks @imliam

**Comment by rverrips** [comment]
Dear @michaeldyrynda
Any comments on this or features that need to be added, or are we just waiting for tailwind v1 to officially drop?

**Comment by michaeldyrynda** [comment]
Was more just waiting for free time to get to this haha

**Pull request merged**